doc/changelog.md

```diff
@@ -40,7 +43,8 @@
 This release contains contributions from (in alphabetical order):
 
 Ali Asadi,
-David Ittah.
+David Ittah,
+Mikhail Andrenkov.
```

Comment:
We usually use alphabetical order of the family names.

Reply:
Gotcha, thanks!

I fixed the issue in the development release notes but it looks like there are a few other places in the changelog where the authors are in the wrong order.